packages/react-native-adapter/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@thirdweb-dev/react-native-adapter",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thirdweb-dev/js.git#main"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/thirdweb-dev/js/issues"
+  },
+  "author": "thirdweb eng <eng@thirdweb.com>",
+  "type": "module",
+  "main": "./dist/cjs/exports/thirdweb.js",
+  "module": "./dist/esm/exports/thirdweb.js",
+  "types": "./dist/types/exports/thirdweb.d.ts",
+  "typings": "./dist/types/exports/thirdweb.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/types/exports/thirdweb.d.ts",
+      "import": "./dist/esm/exports/thirdweb.js",
+      "default": "./dist/cjs/exports/thirdweb.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": ["dist/*", "src/*"],
+  "dependencies": {
+    "@walletconnect/react-native-compat": "^2.13.0"
+  },
+  "peerDependencies": {
+    "typescript": ">=5.0.4",
+    "@react-native-async-storage/async-storage": "*",
+    "@react-native-community/netinfo": "*",
+    "expo-application": "*",
+    "react-native-get-random-values": "*"
+  },
+  "peerDependenciesMeta": {
+    "typescript": {
+      "optional": true
+    },
+    "expo-application": {
+      "optional": true
+    }
```

Comment:
is this truly optional? if so why?

Reply:
I believe you don't need it if you're using react native CLI

Reply:
gotcha makes sense